ai/src/main/java/com/shatteredpixel/shatteredpixeldungeon/ai/AIPixelDungeon.java

```diff
@@ -14,6 +17,10 @@
 public class AIPixelDungeon extends ShatteredPixelDungeon {
     public AIPixelDungeon(PlatformSupport platform) {
         super(platform);
+        GamesInProgress.selectedClass = HeroClass.WARRIOR;
```

Comment:
could you add some comments as to what parts of the codebase you pulled this logic from?

Reply:
I'm not sure if the constructor is where this should live -- are you sure this is better than having it live in the `update` function?